packages/app/src/components/time-series-chart/components/timespan-annotation.tsx

```diff
@@ -127,3 +89,29 @@ export function TimespanAnnotationIcon({
     </svg>
   );
 }
+
+export function HatchedTimespanAnnotationIcon() {
+  return (
+    <svg height="15" width="15">
+      <defs>
+        <pattern
+          id="hatch"
+          width="4"
+          height="4"
+          patternTransform="rotate(-45 0 0)"
+          patternUnits="userSpaceOnUse"
+        >
+          <line
+            x1="0"
+            y1="0"
+            x2="0"
+            y2="5"
+            style={{ stroke: 'grey', strokeWidth: 3 }}
```

Comment:
did you mean `gray`?
```suggestion
            style={{ stroke: 'gray', strokeWidth: 3 }}
```

Reply:
Fixed